jerry-core/ecma/base/ecma-globals.h

```diff
@@ -79,6 +79,7 @@ typedef enum
   ECMA_SIMPLE_VALUE_UNDEFINED, /**< undefined value */
   ECMA_SIMPLE_VALUE_NULL, /**< null value */
   ECMA_SIMPLE_VALUE_NOT_FOUND, /**< a special value returned by ecma_op_object_find */
+  ECMA_SIMPLE_VALUE_NOT_FOUND_AND_STOP, /** not found and not search the proto chain */
```

Comment:
`ecma_op_object_find` use "ECMA_SIMPLE_VALUE_NOT_FOUND" to indicate the prop is not in the object.  While for typedarray, when the prop name is a number, and the prop is not in the own object, it should not search in the prototype chain any more. So I add the "NOT_FOUND_AND_STOP"